kubernetes-azure-python/__main__.py

```diff
@@ -0,0 +1,122 @@
+"""An Azure RM Python Pulumi program"""
+
+import base64
+import pulumi
+from pulumi_azure_native import resources
+from pulumi_azure_native import network
+from pulumi_azure_native import containerservice
+
+# Get some values from the Pulumi configuration (or use defaults)
+# First pull provider-namespaced configuration values
+azure_cfg = pulumi.Config("azure-native")
+azureLocation = azure_cfg.require("location")
+# Second get project-namespaced configuration values
+proj_cfg = pulumi.Config()
+num_worker_nodes = proj_cfg.get_float("numWorkerNodes", 3)
+k8s_version = proj_cfg.get("kubernetesVersion", "1.24.3")
+prefix_for_dns = proj_cfg.get("prefixForDns", "pulumi")
+node_vm_size = proj_cfg.get("nodeVmSize", "Standard_DS2_v2")
+mgmt_group_id = proj_cfg.require("mgmtGroupId")
+ssh_pub_key = proj_cfg.require("sshPubKey")
+
+# Create an Azure Resource Group
+resource_group = resources.ResourceGroup("resource_group",
+    location=azureLocation,
+    resource_group_name="aks-rg"
+)
```

Comment:
```suggestion
resource_group = resources.ResourceGroup(
    "resource_group",
    location=azureLocation,
    resource_group_name="aks-rg"
)
```

Reply:
Should be fixed/correct in 5b8b6ce

Reply:
FWIW, I've been using `black` to format my Python code and it's been working well (I think -- @nimbinatus will let me know if I'm wrong about that!).

Reply:
I think all these Python formatting errors should be corrected now, thanks!